src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -21,8 +21,9 @@ public ScriptExecutionTests(ITestOutputHelper testOutputHelper)
         [Fact]
         public void ShouldExecuteHelloWorld()
         {
-            var (output, _) = ScriptTestRunner.Default.ExecuteFixture("HelloWorld", "--no-cache");
-            Assert.Contains("Hello World", output);
+            ScriptTestRunner.ExecuteFixtureInProcess("HelloWorld", "--no-cache");
+            // var (output, _) = ScriptTestRunner.Default.ExecuteFixture("HelloWorld", "--no-cache");
+            // Assert.Contains("Hello World", output);
```

Comment:
what happened here?

Reply:
Forgot about that one. Just to be able to test in process during development. Fixing